### ISSUE TITLE
User installation of third party plugins

### ISSUE THREAD

**Issue by squinkylabs** [VCVRack/Rack#706]
The manual says on windows to unzip them into  MyDocuments\plugins. But actually you have to unzip them and move them right?

For example, with the AS plugins I download AS-0.5.5-win.zip. If I unzip that in the plugins folder it creates:

plugins\AS-05.5-win\AS\plugin.dll

But VCVRack does not search recursively, so the user needs to move the AS folder to be directly under the plugins folder.

Just want to verify that's what users (at least on windows) need to do to download and install third party plugins.

**Comment by squinkylabs** [comment]
@AScustomWorks : I think it's with all plugins. And yes, I got yours from your github. Pretty sure the same is true of Audible Illusions, as well as my own when packaged by `make dist`. Perhaps it's an artifact of how windows unzips things - it always creates a top level folder whose name is the filename of the zip. If memory serves me, that's what happens when you double click on a zip on a mac, too.

**Comment by AndrewBelt** [comment]
"unzip them into Rack/plugins" means to unzip them to `Rack/plugins`, not unzip them to `Rack/plugins/name-of-zip-basename`. Your unzipping software is incorrectly adding that directory for you.

**Comment by squinkylabs** [comment]
Yes, I verified, too. Mac is fine, it's just windows that does that. I'm going to document this in my GitHub README. (I only tried on Windows-7. I'll try 10 next time I'm at work and remember).

I don't want to start a flame war, but I think there are users of VCV on windows, and the fact that the windows explorer built-in unzip is non standard will be of little comfort to people who can't install their plugins.

(And btw - it's great to see myself in the plugin manager! Thanks for making such a wonderful ecosystem.)
